examples/scripts/script_reslc.py

```diff
@@ -98,39 +98,79 @@ def get_free_port():
     metadata = read_metadata(f_mother_res)
 
     # Coordinates
-    f_lam = mother_dir / "lam.raw"
-    f_phi = mother_dir / "phi.raw"
+    f_lam = mother_dir / "lam.raw" # lon
+    f_phi = mother_dir / "phi.raw" # lat
 
     # Mother SLC
     f_mother_slc = mother_dir / "slave_rsmp_reramped.raw"
 
     # List of SLC
     f_ifgs = list(sorted(stack_dir.rglob("2*/cint_srd.raw")))
-    f_ifgs = f_ifgs[:3]
+    f_h2phs = list(sorted(stack_dir.rglob("2*/h2ph_srd.raw")))
 
     shape = (metadata["n_lines"], metadata["n_pixels"])
     dtype_slc_ifg = np.dtype([("re", np.float32), ("im", np.float32)])
     dtype_lam_phi = np.float32
 
-    # Lazy loading mother SLC and ifg stack
-    mother = sarxarray.from_binary([f_mother_slc], shape, dtype=dtype_slc_ifg, chunks=reading_chunks)
-    ifgs = sarxarray.from_binary(f_ifgs, shape, dtype=dtype_slc_ifg, chunks=reading_chunks)
+    # Lazy loading ifg stack
+    ifgs = sarxarray.from_binary(f_ifgs, shape, dtype=dtype_slc_ifg, chunks=reading_chunks) # Load ifgs
+    ifgs_h2ph = sarxarray.from_binary(f_h2phs, 
```

Comment:
Multi-line formatting here, not for other lines. Suggest to change this one, to be homogeneous.